app/controllers/applicants/omniauth_callbacks_controller.rb

```diff
@@ -6,10 +6,7 @@ class OmniauthCallbacksController < Devise::OmniauthCallbacksController
 
     def true_layer
       if applicant
-        applicant.update!(
-          true_layer_token: token,
-          true_layer_token_expires_at: token_expires_at
-        )
+        import_bank_data
```

Comment:
❓ If this process fails (an exception is raised for example), is the expected outcome to show a 500 page or display an error page to the user stating there was a problem fetching their bank details?

Reply:
Looks like there's a story about showing a message to the user: https://dsdmoj.atlassian.net/browse/AP-173

I'm going to modify this PR to show the raw errors to the user and AP-173 can refine it.